Organic/Organic.php

```diff
@@ -741,7 +741,11 @@ public function syncPost( $post ) {
         $content = \apply_filters( 'organic_post_content', $post->post_content, $post->ID );
         $is_published = \apply_filters( 'organic_post_is_published', $post->post_is_published, $post->ID );
         $published_date = \apply_filters( 'organic_post_publish_date', $post->post_date, $post->ID );
-        $modified_date = \apply_filters( 'organic_post_modified_date', $post->post_modified, $post->ID );
+        $modified_date = \apply_filters(
+            'organic_post_modified_date',
+            \apply_filters( 'get_display_datetime', $post->ID ),
```

Comment:
Not sure why we need another filter here? I think using the `organic_post_modified_date` should be enough
Similar to this: https://github.com/orgnc/api.thedrive.com/blob/04cf6b75fd1ece0e78f54c17a062d6f4fb3facb3/src/wp-content/plugins/empire-integration/empire-integration.php

That's an outdated repo with an old plugin (which was named `empire` at that time) but should give a general idea